release-notes-2.0.0.md

```diff
@@ -1,10 +1,9 @@
-!not-ready-for-release!
 
 #### Version Number
 ${version-number}
 
 #### New Features
-
+- Initial Release
```

Comment:
This is not the initial release of this container, this is version 2.0.0 so this is not correct for the release notes. You should state that the base image was updated to use elasticsearch version 2.4.6